mkosi/__init__.py

```diff
@@ -6417,6 +6432,7 @@ def build_image(
                 reset_machine_id(args, root, do_run_build_script, for_cache)
                 reset_random_seed(args, root)
                 run_finalize_script(args, root, do_run_build_script, for_cache)
+                invoke_fstrim(args, root, do_run_build_script, for_cache)
```

Comment:
Is this correct, you check if the image is cached in the function but you pass for_cache instead of cached as the argument. 

So currently, this only runs when building a cached image. Is that intended? If so, let's name the function argument for_cache instead of cached

Reply:
fixed